test/extensions/EXT/SPV_EXT_relaxed_printf_string_address_space/builtin_printf.ll

```diff
@@ -0,0 +1,41 @@
+; Test that calls to "printf" are mapped to OpenCL Extended instruction "printf"
+; Also ensure that spirv-val can validate format strings in non-constant space
+;
+; Testcase derived from:
+;   #include <sycl/sycl.hpp>
+;   int main() {
+;     sycl::queue queue;
+;     queue.submit([&](sycl::handler &cgh) {
+;       cgh.single_task([] {
+;         __builtin_printf("%s, %s %d %d %d %s!\n", "Hello", "world", 1, 2, 3, "Bam");
+;       });
+;     });
+;   }
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: not llvm-spirv %t.bc -o %t.spv 2>&1 | FileCheck %s --check-prefix=CHECK-WO-EXT
+
+; RUN: llvm-spirv -spirv-text %t.bc -o %t.spt --spirv-ext=+SPV_EXT_relaxed_printf_string_address_space
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv --spirv-ext=+SPV_EXT_relaxed_printf_string_address_space
+; RUN: spirv-val %t.spv
```

Comment:
I'm afraid that the spirv-val step in these new tests will have to be disabled until your SPIRV-Tools fix has been merged and a new SPIRV-Tools release has been made, otherwise the CI will be red.  SPIRV-Tools releases happen every 2 months or so.